02R-tist.qmd

```diff
@@ -28,7 +28,14 @@ Substantial free training is available for R. A few links to get you started are
 
     -   [Efficicient R Programming - Colin Gillespie, Robin Lovelace](https://bookdown.org/csgillespie/efficientR/)
 
-    -   [Bookdown](https://bookdown.org/)
+    -   [quarto](https://quarto.org)
+
+    -   [Tidyverse Style Guide](https://style.tidyverse.org/)
+
+    -   [Best Coding Practices](https://bookdown.org/content/d1e53ac9-28ce-472f-bc2c-f499f18264a3/)
```

Comment:
I think we agreed not to use this link, because many important sections like testing are left blank.